packages/button/docs/IconButton.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import configureIcon from "@kaizen/component-library/icons/configure.icon.svg"
+import * as React from "react"
+import { withDesign } from "storybook-addon-designs"
+import { IconButton } from ".."
+import { CATEGORIES, SUB_CATEGORIES } from "../../../storybook/constants"
+import { figmaEmbed } from "../../../storybook/helpers"
+
+export default {
+  title: `${CATEGORIES.components}/${SUB_CATEGORIES.button}/Icon Button`,
+  component: IconButton,
+  args: {
+    label: "Label",
+  },
+  parameters: {
+    actions: {
+      argTypesRegex: "^on.*",
+    },
+    docs: {
+      description: {
+        component:
+          'import { IconButton } from "@kaizen/button". This IconButton supersedes "@kaizen-draft/button".',
+      },
+    },
+    ...figmaEmbed(
+      "https://www.figma.com/file/eZKEE5kXbEMY3lx84oz8iN/%E2%9D%A4%EF%B8%8F-UI-Kit%3A-Heart?node-id=13555%3A0"
+    ),
+  },
+  decorators: [withDesign],
+}
+
+export const DefaultKaizenSiteDemoIcon = args => (
+  <IconButton {...args} icon={configureIcon} />
+)
+
+DefaultKaizenSiteDemoIcon.storyName = "Default Icon (Kaizen Site Demo)"
```

Comment:
```suggestion
DefaultKaizenSiteDemoIcon.storyName = "Default (Kaizen Demo)"
```

Alt put `Button` in it. Since it's not actually an Icon story.

Reply:
nit/ Don't rename the `(Kaizen Site Demo)`. We can use it to filter out all the stories we embed on the Gatsby site.

Reply:
Hmm...That's an interesting one. Pretty sure we've been renaming the newer ones to be without `Site` because no one else knew that was a thing 🙃

Reply:
Ah. We should stop doing that 😬

Reply:
I mean, as long as it's consistent moving forward it should be okay right? Or is there something set up to look for it?